### ISSUE TITLE
Simulator Build Fails when Importing Swift Package into iOS App with a GRDB dependency

### ISSUE THREAD

**Issue by rnantes** [groue/GRDB.swift#911]
I have a 'MyDatabase' Swift Package Manager Package that has a dependency on GRDB version 5.3.0 and it builds and runs tests fine. However, I have an iOS app 'MyApp' that has a dependency on the 'MyDatabase' package that fails to build for an iOS simulator (compiling for a real device works) with the error message: Command CompileSwift failed with a nonzero exit code. I'm stumped in looking for a solution, changing things like VALID_ARCH didn't seem to work as described here: https://github.com/CocoaPods/CocoaPods/issues/10059, or EXLUDED_ARCH as described here: https://stackoverflow.com/questions/63607158/xcode-12-building-for-ios-simulator-but-linking-in-object-file-built-for-ios/63955114#63955114. Any suggestions would be much appreciated.

Xcode Version: 12.3
macOS Verion: 11.1 Big Sur

**Comment by groue** [comment]
Hello @rnantes. Please provide a minimal reproducible case. I insist on **minimal**.

**Comment by rnantes** [comment]
Thanks for the quick response @groue. I tested importing a minimal GRDB Swift package and it actually was working as expected. Turns out my issue was from a bug in the new Resources support for Swift Package Manager that fails to compile when a resource directory is named 'Resources' as described on [Stackoverflow](https://stackoverflow.com/a/54149978). As the bug was on my side I will close the issue. Thanks Again.
